drivers/led/Kconfig.gpio

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2018 Manivannan Sadhasivam
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config LED_GPIO
+	bool "LED GPIO driver"
+	depends on GPIO
+	help
+	  Enable software driven LED using GPIO pin.
+
+# ---------- LED 0 ----------
+
+config LED0_GPIO
+	bool "Enable LED GPIO 0"
+	depends on LED_GPIO
+	help
+	  Enable LED GPIO 0.
+
+if LED0_GPIO
+
+config LED0_GPIO_NAME
```

Comment:
Can't we use the 'label' to get the instance name?

Reply:
okay.